application/F3DStarter.cxx

```diff
@@ -53,6 +53,8 @@ namespace fs = std::filesystem;
 // This pointer is used to retrieve the interactor in case an OS signal is handled
 f3d::interactor* GlobalInteractor = nullptr;
 
+static constexpr int EVENT_LOOP_TIME = 30;
```

Comment:
can we just move it in `F3DStarter::Start` and make it non-static?

Reply:
its used by playInteraction too